pkg/features/kube_features.go

```diff
@@ -232,12 +232,15 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 		{Version: version.MustParse("0.10"), Default: false, PreRelease: featuregate.Alpha},
 	},
 	TASProfileMostFreeCapacity: {
+		{Version: version.MustParse("0.10"), Default: false, PreRelease: featuregate.Alpha},
 		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Deprecated},
 	},
```

Comment:
Without this I got an error 

```
panic: feature "TASProfileMostFreeCapacity" introduced as deprecated must provide a 1.0 entry indicating initial state
```

Reply:
Can you specify 0.11 for Alpha too? Saying this way 0.10 could be misleading

Reply:
So instead of marking it as Deprecated, we should mark it as Alpha, correct?

Reply:
Maybe we should set it as Alpha for 0.11 and Deprecated in 0.12? WDYT?

Reply:
Actually, a little bit different.

```
panic: feature "TASProfileMostFreeCapacity" lists version transitions in non-increasing order (0.11 <= 0.11)
```

Reply:
It looks like this error message came from the below conditional
https://github.com/kubernetes/kubernetes/blob/5be79417862d29aecfadb2735cc42da64dff3ba7/staging/src/k8s.io/component-base/featuregate/feature_gate.go#L444-L447

So, this hack seems to be needed...

Reply:
I'm ok with the trick